microsoft-edge/extensions-chromium/index.md

```diff
@@ -119,6 +119,8 @@ For example, the [Microsoft Editor: Spelling & Grammar Checker](https://microsof
 
 The Microsoft Edge Add-ons team reviews each extension for adherence to [Best Practices for extensions](./developer-guide/best-practices.md) before granting the extension a **Featured** badge.  Badged extensions are curated solely for the purpose of providing Microsoft Edge users with secure browsing tools and high-quality experiences.  Developers cannot pay to receive a **Featured** badge for an extension.
 
+To help protect your browser settings, extensions that don't comply with EAS best practices are not badged currently. 
```

Comment:
expand acronym EAS

Reply:
Edge Add-Ons Store
```suggestion
To help protect your browser settings, extensions that don't comply with Edge Add-Ons Store best practices are not badged.
```
no need to say "currently", usually, b/c could put that on every statement in docs